js/index.js

```diff
@@ -1,7 +1,87 @@
 // Iteration 1: Names and Input
+let hacker1 = "Marcel";
+console.log(`"The driver's name is ${hacker1}."`);
+
+let hacker2 = "Mia";
+console.log(`"The navigator's name is ${hacker2}."`);
 
 
 // Iteration 2: Conditionals
+if (hacker1.length > hacker2.length) {
+  console.log(
+    `The driver has the longest name, it has ${hacker1.length} characters.`
+  );
+} else if (hacker1.length < hacker2.length) {
+  console.log(
+    `It seems that the navigator has the longest name, it has ${hacker2.length} characters.`
+  );
+} else {
+  console.log(
+    `Wow, you both have equally long names, ${hacker1.length} characters!`
+  );
+}
+
+
+// Iteration 3: Loops
+
+hacker1 = hacker1.toUpperCase();
+let letter;
+let driverCapitalized = "";
```

Comment:
you do not need to have 2 variables here you can just work directly with driverCapitalized, and add the letters with a space to this string.